crates/nu-command/src/filesystem/ls.rs

```diff
@@ -131,9 +131,13 @@ impl Command for Ls {
                         Err(e) => {
                             if e.kind() == ErrorKind::PermissionDenied
                                 || e.kind() == ErrorKind::Other
+                            // on windows this is "The process cannot access the file
+                            // because it is being used by another process. (os error 32)"
+                            || e.raw_os_error() == Some(32)
```

Comment:
I was wondering if we should just be more resilient here and always list the filename, regardless of the ErrorKind.

Reply:
well, this part wants to return `metadata`. i looked at how to create it like `fs::Metadata { blah }` but didn't see anything like it. i'm fine with closing this for a better alternative. i was just looking for something to not fail.